spec/support/seed_db.rb

```diff
@@ -1 +1,23 @@
-# load "db/seeds.rb"
\ No newline at end of file
+require "database_cleaner"
+
+DatabaseCleaner.clean_with(:truncation, :only => ["airports", "flights"])
+
+airports_list = [
+  { name: "Murtala Muhammed International Airport", country: "Nigeria", city: "Lagos", code: "MF1" },
+  { name: "Nnamdi Azikwe International Airport", country: "Nigeria", city: "Abuja", code: "MF2" },
+  { name: "Dubai International Airport", country: "Dubai", city: "Garhoud", code: "MF10" },
+  { name: "Manchester Airport", country: "England", city: "Manchester", code: "MF4" },
+  { name: "Amsterdam Airport Schiphol", country: "Holland", city: "Haarlemmermeer", code: "MF16" }
```

Comment:
Indent the right bracket the same as the start of the line where the left bracket is.